### ISSUE TITLE
Modify the artifact manager to get and use torrents from the DHT to get artifacts from the Pyrsia network when the artifact manager is asked for an artifact that is not in the local repository.

### ISSUE THREAD

**Issue by mgrand** [pyrsia/pyrsia#282]
This is to be done after #281.

Modify the artifact manager to so that when there is a request to pull an artifact that is not in the local repository, it looks for a torrent in the DHT that it can use to download the artifact. If it finds such a torrent, it asks Synapse to download it.

**Comment by efrisch** [comment]
Closing due to bittorrent stories being out of scope / replaced with bitswap.
